### PR TITLE
XML to JSON Proxy

### PR DESCRIPTION
**Pull request by jimmydief**
URL parameter must be UTF-8 encoded to work correctly.

screen shot 2015-04-06 at 5 15 11 pm

### CODE REVIEW

**Comment by jimmydief** [comment]
@shankari @Wuhandson here's the proxy. Super simple and should simplify things a lot.

File: requirements.txt [hunk 1]
```diff
@@ -7,4 +7,7 @@ requests==2.5.1
 six==1.9.0
 utm==0.3.1
 wsgiref==0.1.2
-google-api-python-client==1.1
\ No newline at end of file
+google-api-python-client==1.1
+jsonpickle
```
Comment (shankari) [inline requirements.txt]: nit: This change does not require jsonpickle or python-crontab.
Also, don't you need to add urllib2?

**Comment by shankari** [comment]
One minor nit: otherwise looks good.

File: requirements.txt (hunk 1 above)
Comment (jimmydief) [inline requirements.txt]: @shankari good catch

**Pull request merged**